money-monkey-web/drizzle.config.ts

```
@@ -0,0 +1,11 @@
+import "@env";
+import { type Config } from "drizzle-kit";
+
+export default {
+  schema: "./app/lib/db/schema.ts",
+  dialect: "postgresql",
+  dbCredentials: {
+    url: process.env.POSTGRES_URL!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)